docs/readthedocs/source/doc/LLM/Overview/KeyFeatures/finetune.md

```diff
@@ -10,10 +10,12 @@ We also support finetuning LLMs (large language models) using QLoRA with BigDL-L
 
 To help you better understand the finetuning process, here we use model [Llama-2-7b-hf](https://huggingface.co/meta-llama/Llama-2-7b-hf) as an example.
 
-**Make sure you have prepared environment following instructions [here](../install_gpu.html). First of all, you need to import `intel_extension_for_pytorch` to run on Intel GPUs**:
+**Make sure you have prepared environment following instructions [here](../install_gpu.html).**
 
-```python
-import intel_extension_for_pytorch as ipex
+```eval_rst
+.. note::
+
+   If you are using an older version of ``bigdl-llm`` (specifically, older than 2.5.0b20240104), you need to manually add ``import intel_extension_for_pytorch as ipex`` at the beginning of your code.
```

Comment:
In the following code block, we still import ipex first, should we also remove it ? (same question in inference page)

Reply:
Thanks a lot for the comments :) has removed.